script/upgrade/Mode-100-to-100.s.sol

```diff
@@ -0,0 +1,351 @@
+import {Script} from "forge-std/Script.sol";
+import {Test, console2 as console} from "forge-std/Test.sol";
+
+import {Multisig} from "@aragon/multisig/Multisig.sol";
+import {IDAO} from "@aragon/osx/core/dao/IDAO.sol";
+
+import {GaugesDaoFactory, GaugePluginSet, DeploymentParameters, Deployment, TokenParameters, DAO, IGaugeVote} from "@factory/GaugesDaoFactory.sol";
+import {VotingEscrow, Lock, QuadraticIncreasingEscrow, ExitQueue, SimpleGaugeVoter, SimpleGaugeVoterSetup, ISimpleGaugeVoterSetupParams} from "@voting/SimpleGaugeVoterSetup.sol";
+import {SimpleGaugeVoterV1_1_0} from "@voting/SimpleGaugeVoter_v1_1_0.sol";
+
+import {Upgrades} from "@foundry-upgrades/LegacyUpgrades.sol";
+import {Options} from "@foundry-upgrades/Options.sol";
+
+contract UpgradeModeTo110 is Script, Test {
+    /////////////////////////////////////////////
+
+    /////////////////////////////////////////////
+
```

Comment:
I'd put `vm.envString("NETWORK"), vm.envAddress("FACTORY_ADDRESS"), vm.envAddress("SIGNER_ADDRESS")` as internal variables here. It lets us immediatelly see what env variables are needed.